<!DOCTYPE html>
<html>
  <head>
    <meta charset="UTF-8">
    <title>Rakesh's Project - Genomic Data Analytics in Parkinson’s Disease</title>
    <link rel="stylesheet" href="../style.css">
      <style>
		/* Set default font family and size */
body {
  font-family: Arial, sans-serif;
  font-size: 16px;
}

/* Style the page header */
h1 {
  text-align: center;
  font-size: 32px;
  color: #2b2d42;
}

/* Style the section headers */
h2 {
  font-size: 24px;
  color: #2b2d42;
  margin-top: 40px;
  margin-bottom: 20px;
    padding-left: 20px;
    padding-right: 20px;

}

/* Style the paragraph text */
p {
  line-height: 1.5;
  margin-bottom: 20px;
    padding-left: 20px;
    padding-right: 20px;
    text-align: justify;
}

/* Style the strong text */
strong {
  font-weight: bold;
}

/* Style the keyword text */
strong + p {
  font-size: 14px;
  color: #7b7f9e;
}

/* Center align the date section */
div {
  text-align: center;
  margin-bottom: 40px;
    align-items: center;
}

/* Style the page footer */
footer {
  margin-top: 40px;
  text-align: center;
  color: #7b7f9e;
}
.text-center {
  text-align: center;
}

.text-center p:not(.no-center) {
  text-align: left;
}


    </style>
  </head>
  <body>
  <header style="height: 55px">
  <h1>Rakesh's Portfolio</h1>
  <nav>
    <ul style="text-align: center; display: flex;">
      <li style="margin-right: 1em;"><a href="../pilot.html">Home</a></li>
      <li><a href="../contact.html">Contact Me</a></li>
    </ul>
  </nav>
</header>
  <br>
  <br>
  <br>
  <br>
    <h1>Project 1: Genomic Data Analytics in Parkinson’s Disease</h1>

<div class="text-center">
    <br>
  <p><strong>Jan 2021 - Apr 2021</strong></p>
  <p>Associated with Sathyabama Institute of Science &amp; Technology, Chennai</p>
</div>

    <h2>Project Details:</h2>
    <p>I developed a comprehensive workflow that incorporates various tools for the accurate assessment and prediction of SNP-level severe damages. The workflow involves the evaluation of SNP effects, secondary structural studies, homology modeling, visualization, and stability studies, all of which are critical in identifying dangerous non-reported mutations and variants. The analysis has been carried out on three genes, namely SNCA, LRRK2, and PARK7, to ensure a thorough and reliable evaluation of the SNPs. This workflow is an essential tool for researchers and scientists who are interested in identifying and predicting potentially harmful SNPs.</p>

    <h2>Abstract:</h2>
    <p>Parkinson's is a progressive, tremor-characterized, neurodegenerative disease. Parkinson’s being second most affecting neuro-related disease after Alzheimer’s and most adults develop the disease at around 60 years of age, but some have an early-onset of disease and forms of Parkinson's are inherited where certain forms have been related to unique gene mutations that can be genetic or environment associated but are not understood completely. This study concentrates on existing data from various repositories to find and predict SNP level severe damages in genes of Parkinson’s disease especially in 3 of the important genes SNCA, LRRK2, PARK7. SNPs of the retrieved genes were analyzed by in-silico tools like SIFT, SNAP, iMutant suite, MetaSNP, Mutation Taster; Algorithms like CADD, DANN and various functions to determine the changes in structural and functional aspects. Sorting out only for harming and deleterious SNPs in all the 3 genes finally landed with 11 variants in PARK7, 4 variants in SNCA and 5 variants in LRRK2 which actually has an impact on functional level from initial set of 1000+ variant counts. Homology modeling using Swiss Model and validation using PROCHECK was done and further visualization for structure related studies were performed in HOPE server. Further, Stability related studies were done using all the data from various computational tools from which variants like R156, P73 in PARK7; E123K in SNCA and R1628, L1795 in LRRK2 were found to be damaging to great extent. Furthur extensive analysis required onto these variants and variants A107P in PARK7, K96I in SNCA and N2261L in LRRK2 had a positive impact on the protein stability which also requires separate study on how mutation increases the stability of diseased protein.</p>
    <p><strong>Keywords:</strong> Parkinson’s Disease, SNPs in Parkinson, Mutations in Parkinson’s, Early onset of Parkinson.</p>
  <p>Parkinson’s disease is a neurodegenerative disorder that affects millions of people worldwide. The disease is caused by the death of dopamine-producing neurons in the brain, leading to tremors, stiffness, and difficulty with movement. While there is no cure for Parkinson’s disease, early diagnosis and treatment can help manage symptoms and improve quality of life. Genomic data analytics has emerged as a promising tool for identifying genetic risk factors associated with Parkinson’s disease.</p>

  <p></p>

    <h2>TL;DR:</h2>
    <p>Developed a workflow utilizing tools for evaluating SNP effects, conducting secondary structural analysis, creating homology models, visualizing data, and conducting stability studies to identify potentially harmful non-reported mutations and variants. Applied this workflow to analyze 3 genes: SNCA, LRRK2, and PARK7.</p>
  </body>
</html>